hadoop-hdds/dev-support/checkstyle/suppressions-skip-imports.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+<!DOCTYPE suppressions PUBLIC
+    "-//Checkstyle//DTD SuppressionFilter Configuration 1.1//EN"
+    "https://checkstyle.org/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+  <suppress checks="JavadocPackage" files="[\\/]src[\\/]test[\\/].*"/>
+  <suppress checks="JavadocPackage" files="src/main/java/org/apache/hadoop/io/erasurecode/rawcoder"/>
+  <suppress checks="IllegalImport" files="src/test/java/org/apache/hadoop/fs/contract/.*java"/>
+  <suppress checks="MissingJavadocType" files="src/test/java/org/apache/hadoop/fs/contract/.*java"/>
+  <suppress checks="IllegalImport" files="src/test/java/org/apache/hadoop/tools/contract/.*java"/>
+  <suppress checks="CustomImportOrder" files=".*"/>
+  <suppress checks="Header" files=".*"/>
```

Comment:
Please add a comment in `suppressions.xml` mentioning that it is duplicated temporarily in this new file.